.npmrc

```diff
@@ -1,3 +1,4 @@
 ; adding this as npm 7 automatically installs peer dependencies but pnpm does not
 auto-install-peers=true
 strict-peer-dependencies=false
+enable-pre-post-scripts=true
```

Comment:
I really liked that `pnpm` introduced predictability by default and doesn't enable pre- and post-scripts.

What would you think about instead of enabling this, changing `build` to `prebuild && ...` (or using a different name for `prebuild` and doing the same)?

Reply:
I pushed up changes. I assume you mean like that?